### ISSUE TITLE
Test question

### ISSUE THREAD

**Issue by tzynzs** [Anirudh257/strm#8]
Hi，It's nice to see the work you do,but I'm having a little difficulty.
I use your pre-trained model to test the hmdb51.What if I only want to test one class video? Set way to 1, but then loss is 0 and the accuracy is 1.

**Comment by Anirudh257** [comment]
@tzynzs thanks for your interest in my work. I am not understanding your question. Could you rephrase it better. You want to test on 1 video sample or query-class=1 video?

**Comment by tzynzs** [comment]
> @tzynzs thanks for your interest in my work. I am not understanding your question. Could you rephrase it better. You want to test on 1 video sample or query-class=1 video?

thank you for your reply.I want to test 1 video to see the accuracy of the prediction.Either hmdb's or my own data.How do i do it?

**Comment by Anirudh257** [comment]
Okay, you will need to modify the test function a bit. 

1) Use a pre-trained model and load it before testing.

2) Look into the paper and sample K frames for your video. 

3) Check the labels of HMDB /Your dataset and see if they properly match with the dataset used by the pre-trained model. If not, the predictions won't make sense.

**Comment by tzynzs** [comment]
Thanks for your detailed answer!
I understand what you mean.These videos are based on datasets to predict.
If I trained a model using hmdb or something,and how do I use this model to predict an unlabeled new video?

**Comment by Anirudh257** [comment]
@tzynzs you can just feed the video to the model, and look up the data feeder to see the classes included. If it is an unlabeled video, only perform testing with the pretrained weights. It will require some understanding of the https://github.com/Anirudh257/strm/blob/main/video_reader.py

**Comment by tzynzs** [comment]
Okay,I get it.Thank you for your help!

**Comment by Anirudh257** [comment]
Your welcome.
